andela/Titan-Devs
# #163518680 Integrate Coveralls Code Coverage Service

#### What does this PR do?
Integrate code coverage service.
#### Description of Task to be completed?
- Install and configure Coveralls code coverage service
- Install and configure nyc as code coverage reporter
#### How should this be manually tested?
run command for test ``` yarn run test``` and check your terminal to see the table containing Coveralls code coverage report. 
#### What are the relevant pivotal tracker stories?
#163518680
#### Screenshots 
<img width="626" alt="screen shot 2019-02-05 at 12 50 24" src="https://user-images.githubusercontent.com/20681465/52268646-b99b7a00-2944-11e9-943b-5c4019035864.png">
